F21/smedipally/.py

```diff
@@ -0,0 +1,95 @@
+def LowerCaseConverter(word,lang):
+
+    lang=lang
```

Comment:
The parametres can see the variables inside the function without the assignment of these variables

Reply:
Okay! I will check this out!